index.html

```diff
@@ -36,6 +36,20 @@
       </div>
     </div>
 
+    <div id="call-for-proposals" class="sponsors">
+      <section>
+        <h2>Call for Proposals is Now Open</h2>
+
+        Rustfest has 8-10 regular speaking slots. Rust is a new language so all proposals are welcome. If you have been working on something for only a few months, but it is interesting, then apply!
```

Comment:
I think we can actually remove "new" here.

This will be our main call to action on the CFP so maybe we can say something like:

> We are currently seeking proposals for 8-10 topics. Doing something fascinating? Why not share it with the community. New speakers and creative topics welcome. 
>
> Curious what it's like? Check out videos from [Berlin](https://www.youtube.com/watch?v=fI4RG_uq-WU&list=PL85XCvVPmGQh8nWR_Z-fTmPGsUWuzb-dn) and [Kyiv](https://www.youtube.com/watch?v=AHprJNUCgQ0&list=PL85XCvVPmGQhvs1Rnet_24B-AI3YSM2YG).